### PR TITLE
Fixes dead PKborgs having active dampener fields

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

**Comment by Militaires** [comment]
150 googolplexianths of testing

**Comment by Repukan** [comment]
Kevinz000 more like Kevinz150

File: code/game/objects/items/robot/robot_items.dm [hunk 1]
```diff
@@ -582,6 +582,10 @@
 	process_recharge()
 	process_usage()
 	update_location()
+	if(isliving(host))
```
Comment (AnturK) [inline code/game/objects/items/robot/robot_items.dm]: If cyborgs don't disable their modules on death, they should. Make this generic hook please.

**Comment by kevinz000** [comment]
just going to put "i need to make on_death for modules" here so i don't forget tomorrow.

**Comment by lzimann** [comment]
you forgot

**Comment by kevinz000** [comment]
fugg

**Comment by kevinz000** [comment]
give me uh 8 hours here tyvm guys

**Pull request merged**